tracer-core/src/main/java/org/zalando/tracer/PhraseGenerator.java

```diff
@@ -0,0 +1,469 @@
+package org.zalando.tracer;
+
+/*
+ * ⁣​
+ * Tracer
+ * ⁣⁣
+ * Copyright (C) 2015 Zalando SE
+ * ⁣⁣
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ​⁣
+ */
+import com.google.common.annotations.VisibleForTesting;
+import java.security.SecureRandom;
+import java.util.function.IntUnaryOperator;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * This generator is inspired by Docker container name generator
+ *
+ * @see https://github.com/docker/docker/blob/master/pkg/namesgenerator/names-generator.go
```

Comment:
The `@see` tag doesn't support links, you need to wrap it into an anchor: `<a href="..">names-generator.go</a>`